.github/workflows/publish.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           node-version: 16
           registry-url: https://registry.npmjs.org/
 
+      - run: sed -i "s/development/$(jq  -r '.version' package.json)/" src/meta.js
```

Comment:
How do you feel about moving this into the [`prepublishOnly` npm lifecycle script](https://docs.npmjs.com/cli/v8/using-npm/scripts)? When run from in there, you should be able to do `sed -i "s/development/$npm_package_version/" src/meta.js` because that environment var is available in that context.